packages/plugin-compiler/src/test/kotlin/io/realm/compiler/GenerationExtensionTest.kt

```diff
@@ -92,25 +93,26 @@ class GenerationExtensionTest {
     }
 
     @Test
+    // FIXME Do we actually want to fail at compile time if there is not schema?
```

Comment:
Since `schema` is a required parameter, it seems rather hard to get into this state. I'm not sure I see a use case for creating a Realm with no schema, but on the other hand, I'm not sure it is worth it to check for the case.

IMO shouldn't we just re-enable this test and maybe change the name to to `define schema in external variable` (or something like that).